docs/cli/edgedb_toml.rst

```diff
@@ -0,0 +1,44 @@
+.. _ref_cli_edgedb_toml:
+
+===========
+edgedb.toml
+===========
+
+The ``edgedb.toml`` file is a key component of EdgeDB's project system. If this
```

Comment:
Do you think it's helpful to link out to any TOML format documentation like https://toml.io/en/v1.0.0 ?

Reply:
I found a note with a link to TOML docs in the projects reference, so I just copied that one over. I did like your target better, so I changed it from targeting the TOML quickstart to what you suggested.